### DISCUSSION TITLE
Any api for cluster failover with takeOver option

### DISCUSSION THREAD

**Discussion by SreedharReddyKallu** [redis/lettuce#2269] [Q&A] [answered]
RedisClusterAsyncCommands is having api with force failover and soft failover but didn't find option for takeOver.
Redis documentation for same : https://redis.io/commands/cluster-failover/

**Comment by mp911de** [comment, marked as answer]
Currently not, feel free to submit a pull request.

**Reply by SreedharReddyKallu** [reply]
@mp911de created pr https://github.com/lettuce-io/lettuce-core/pull/2358 .
Thanks
